src/hardware/pwm.rs

```diff
@@ -149,7 +147,7 @@ impl Pwm {
         fn set_pwm_channel<P: PwmPin<Duty = u16>>(channel: &mut P, duty: f32) -> Result<(), Error> {
             let value = (duty * Pwm::MAX_DUTY as f32) as i32;
             defmt::info!("duty: {:?}", duty);
-            if !(0..Pwm::MAX_DUTY as _).contains(&value) {
+            if !(0..(Pwm::MAX_DUTY + 1) as _).contains(&value) {
```

Comment:
I think

```suggestion
            if !(0..=Pwm::MAX_DUTY as _).contains(&value) {
```

should work.

Reply:
:+1: